money/contrib/django/models/fields.py

```diff
@@ -256,6 +256,12 @@ def formfield(self, **kwargs):
         defaults.update(kwargs)
         return super(MoneyField, self).formfield(**defaults)
 
+    @property
+    def validators(self):
+        # Hack around the fact that we inherit from DecimalField but don't hold
+        # Decimals. The real fix is to stop inheriting from DecimalField.
+        return []
```

Comment:
Actually, I confirmed that (on Postgres at least) the amount will be stored in a `numeric` column, so  the database will not store messed up data. Not ideal, but an OK compromise for the time being